Yosemite/Yosemite/Base/SessionManagerProtocol.swift

```diff
@@ -23,10 +23,6 @@ public protocol SessionManagerProtocol {
     ///
     var defaultStoreID: Int64? { get set }
 
-    /// URL of default store
-    ///
-    var defaultStoreURL: String? { get }
```

Comment:
I think we can also remove [this](https://github.com/woocommerce/woocommerce-ios/blob/trunk/Networking/Networking/Settings/Credentials.swift#L43C17-L43C42) `Credentials.hasPlaceholderSiteAddress()` method, which isn't used anywhere now.

Reply:
Good point - removed in 97d8cd3d78aa9357ae22f64470794e4564974238.